src/tonic/api/conversions.rs

```diff
@@ -98,6 +98,16 @@ impl From<segment::types::ScoredPoint> for ScoredPoint {
     }
 }
 
+impl From<collection::operations::types::Record> for RetrievedPoint {
+    fn from(record: collection::operations::types::Record) -> Self {
```

Comment:
Maybe import `Record`? It is a bit hard to read

Reply:
https://github.com/qdrant/qdrant/pull/343/commits/46c9ad4847be28638f9bddf031d206cafc0b2f46

Reply:
we agreed here, that internal types should be used with full-path specification in this module for avoiding confusion and for consistency

Reply:
https://github.com/qdrant/qdrant/pull/343/commits/b85688da5e258364bec732c66033623485319e61